utam-compiler/src/main/java/utam/compiler/grammar/UtamPageObject.java

```diff
@@ -133,7 +122,7 @@ final class UtamPageObject {
     this.rootLocator = selector == null ? null : selector.getLocator();
     this.description = processRootDescriptionNode(descriptionNode);
     if (metadata != null && !metadata.isObject()) {
-      throw new UtamCompilerIntermediateError(907);
```

Comment:
sorry for removing code 907, I added standard validation to check for empty object to align with JS